### PR TITLE
test: add json logs test environment

### PR DESCRIPTION
**Pull request by smira**
Add an option to `talosctl cluster create` to start a JSON log receiver, and enabled it optionally.

Enable in `integration-qemu`.

See #9510

### CODE REVIEW

File: website/content/v1.9/reference/cli.md [hunk 1]
```diff
@@ -209,6 +209,7 @@ talosctl cluster create [flags]
       --with-debug                               enable debug in Talos config to send service logs to the console
       --with-firewall string                     inject firewall rules into the cluster, value is default policy - accept/block (QEMU only)
       --with-init-node                           create the cluster with an init node
+      --with-json-logs                           enable JSON logs sender and receiver
```
Comment (frezbo) [inline website/content/v1.9/reference/cli.md:212]: just receiever only right?

Comment (smira) [reply website/content/v1.9/reference/cli.md:212]: both, as it patches Talos itself

Comment (smira) [reply website/content/v1.9/reference/cli.md:212]: probably I can reword it to make it more obvious

Comment (smira) [reply website/content/v1.9/reference/cli.md]: updated wording

**Review by DmitriyMV — APPROVED** [review]
